packaging/targets/trusty/debian/install

```diff
@@ -0,0 +1 @@
+/usr/share/nginx/html/*
```

Comment:
That's a good suggestion. Right now it get installed to /usr/share/nginx/html/openhim/ so that is doesn't conflict. But do you think that installing to its own place is better?